src/cmd/index.js

```diff
@@ -6,7 +6,8 @@ import type {
 import type {LintCmdParams, LintCmdOptions} from './lint.js';
 import type {CmdRunParams, CmdRunOptions} from './run.js';
 import type {MultiExtensionRunner} from '../extension-runners/index.js';
-import type {SignParams, SignOptions, SignResult} from './sign.js';
+import type {SignParams, SignOptions} from './sign.js';
+import type {SignResult} from '../util/submit-addon.js';
```

Comment:
can we make sign.js to re-export the SignResult type that is already importing from util/submit-addon.js?